.github/workflows/pr-naming.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull Request naming convention
+# This workflow is triggered on pushes to the repository.
+on:
+  pull_request:
```

Comment:
I think this must be a pull_request_rather if the workflow uses a secret and should be exposed to forks

Reply:
We have something dimilar in t3: https://github.com/t3-oss/create-t3-app/blob/next/.github/workflows/pr-labeler.yml

Reply:
it seems to work 🤷 

try renaming the PR

Reply:
Im not on a form 😎

Reply:
ohhhhh,

Reply:
the docs seem diff for both but i almost wonder should we use the one you're using on T3. seems more popular 🤣

Reply:
https://github.com/trpc/trpc/pull/4380